odl/discr/discretization.py

```diff
@@ -543,6 +543,13 @@ def __init__(self, space, data):
         assert isinstance(space, DiscretizedSpace)
         DiscretizedSetVector.__init__(self, space, data)
 
+    def __ipow__(self, p):
+        """Implement ``self **= p``."""
+        # Falls back to `LinearSpaceVector.__ipow__` if `self.ntuple`
+        # has no own `__ipow__`. The fallback only works for integer `p`.
```

Comment:
The way we implemented it, `__pow__` calls `__ipow__` on a copy, so `__ipow__` has the last word. If an `ntuple` overrides `__pow__`, this `__ipow__` method will never be called (unless that new implementation calls it). If an `ntuple` overrides `__ipow__`, all good. If not, we get `LinearSpace.__ipow__`. I don't see a logical error in the comment.